packages/admin-ui/src/combobox/combobox-popover.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import { ComboboxPopover as BaseComponent } from 'ariakit/combobox'
+import { createComponent, useElement, tag } from '@vtex/admin-ui-react'
+
+import * as style from './combobox.style'
+import { messages } from './combobox.i18n'
+import { ComboboxItem } from './combobox-item'
+import { Paragraph } from '../components/Paragraph'
+import { Text } from '../components/Text'
+import type { ComboboxState } from './combobox.state'
+import { Spinner } from '../components/Spinner'
+import { Button } from '../components/Button'
+import { useMessageFormatter } from '../i18n'
+
+export const ComboboxPopover = createComponent<typeof BaseComponent, Props>(
```

Comment:
At first, I thought we had an Admin UI BaseComponent, maybe adding Ariakit prefix can help the readability
Something like this:
```suggestion
export const ComboboxPopover = createComponent<typeof AriakitComboboxPopover, Props>(
```
Just a nitpick suggestion, feel free to accept

Reply:
:done: